senior_project_dea-main/src/components/Game.js

```diff
@@ -1,218 +1,49 @@
                     </p>
-                    <div id="search-container-3">
-                    <input type="text" placeholder="Email..." id="email"></input>
-                    <input type="text" placeholder="Password..." id="password"></input>
-                    <button className="button" onClick={submit3}>Login</button>
-                    </div>                    
-                </Tab.Pane>
-                <Tab.Pane eventKey="fourth">                    
-                    <p>This question is about input sanitization to prevent things such as Cross-Site Scrypting attacks. Imagine an attacker types in a URL that has some 
-                        script to run some malicious function into a browser (Ex: http://www.testwebsite.com/accdashboard.html#context=&lt;script&gt;SomeFunction(somevariable)&lt;/script&gt;). 
-                        A secure browser with input sanitation would change a malicious input by replacing, removing, or escaping characters. Manipulate this input just as a secure 
-                        browser would so that the script is not run as code, but interpreted as data and type your answer into the input field.
-                     </p>
-                     <div id="search-container-4">
-                    <input type="text" placeholder="Type in answer..." id="search-4"></input>
-                    <button className="button" onClick={submit4}>Submit</button>
-                    </div>                    
-                </Tab.Pane>
-                <Tab.Pane eventKey="fifth">                    
-                    <p>This question focuses on decrypting some text to make sense of it. In this case we have used a Caeser Cypher to encrypt the text. You have to figure out 
-                        what our shifting key is to decode the text. Go to this website (https://www.dcode.fr/caesar-cipher) and decrypt this text 
-                        (Nylha qvi, fvb zbjjlzzmbssf kljyfwalk aopz tlzzhnl! Aol zljyla rlf pz: BMJZNhavyzJfilyzljbypaf) to get the secret key
-                        key and type it into the input field to receive credit. Hint: The key is a number between 1 and 10.
-                    </p>
-                    <div id="search-container-5">
-                    <input type="text" placeholder="Type in secret key..." id="key"></input>
-                    <button className="button" onClick={submit5}>Submit</button>
-                    </div>                    
-                </Tab.Pane>
-            </Tab.Content>
-            </Col>
-        </Row>
-        </Tab.Container>
-    </div>
+                    <a href="./gameAdventure" className="btn btn-primary">
+                        Click Here
+                    </a>
+                    </div>
+                </div>
+            </section>
+            </div>
+        </div>
+    </div>   
   );
 }
```

Comment:
Just for reference, I think we're planning to move most of the traditional game questions into the database as just traditional questions.
But I think there's one that can't be transferred that we can put in GameTraditional.js